scripts/play_vk

```diff
@@ -21,13 +21,14 @@
 # An executable copy of 'play_vk' in your contest directory (containing the
 # logcfg.dat file) can be adapted and has precedence over the standard install
 
+# accept SIGTERM and SIGINT signals and kill play command as soon as possible
```

Comment:
For me using `-s INT` gives a slight (around 0.5 s) delay, before stopping playback.
`-s KILL` stops playback immediately.

Reply:
Sure SIGKILL kills the execution of the script directly by the kernel while SIGINT will be catched by the script and sent to the play command which in turn has to catch the signal and then stops the sound.

I fear the downside of SIGKILL will be that the script gets no chance to do an work you may want it to do AFTER playing the sound. I will check it in the evening.

Comparing it with stopping a CW message the delay for SSB is shorter here.

Reply:
After a quick check I have to confirm some of my fears about the use of SIGKILL. It can not used in stop_vk(). 

But we can use it in the trap line inside the play_vk command. That seems to fix the delay but keeps the script running.

Reply:
The latest play_vk and code works just fine for me.